sentences/nb/weather_HassGetWeather.yaml

```
@@ -0,0 +1,10 @@
+language: nb
+intents:
+  HassGetWeather:
+    data:
+      - sentences:
+          - "Hvordan [s]er været[ ut][ i dag]"
+      - sentences:
+          - "Hvordan [s]er været[ ut] i <navn> [ i dag]"
```

This does not handle `på`, only `i`. I belive this will fix the issue:
```suggestion
          - "Hvordan [s]er været[ ut] <i_på> <navn> [ i dag]"
```

---

Yes, you're absolutely right! Forgot about that option. I'll add it in.